node/builder.go

```diff
@@ -63,36 +62,36 @@ type Config struct {
 	Clock       clock.Clock
 }
 
-// ConfigOpt is a configuration option for a filecoin node.
-type ConfigOpt func(*Config) error
+// BuilderOpt is an option for building a filecoin node.
+type BuilderOpt func(*Builder) error
 
 // OfflineMode enables or disables offline mode.
-func OfflineMode(offlineMode bool) ConfigOpt {
-	return func(c *Config) error {
+func OfflineMode(offlineMode bool) BuilderOpt {
+	return func(c *Builder) error {
```

Comment:
Nit: A little bit weird to have these receivers named `c` and `nc` now.

Reply:
Yea... I guess if you are gonna go for single char variable names might as well just call it all `x`..